src/documents/community-day.md

```diff
@@ -11,79 +11,79 @@ Registration at the conference is not a prerequisite for participation in the Co
+**What to bring:**
 
+Make sure you wear layers that will give you comfort as the temperature up on top of the mountain is usually 10°C than in the Hobart CBD. Pack for wind, rain and sun. Other things you need include:
 
-### **_MONA self-guided adventure_**
-![MONA](/imgs/Mona-trip2.png)
+* Sturdy walking shoes
+* Sunscreen  
+* Water  
+* Snacks  
+* Lunch - or you can purchase at the C3 cafe before we leave.  
 
-The Museum of Old and New Art -MONA- is truly an experience. Gather a group of friends and catch the ferry to MONA and see the sights on the way there. Entry to MONA will cost $39 if you aren’t a Taswegian. [Check out the website](https://mona.net.au/visit) to see if there are any special additional exhibitions to buy tickets to. 
 
-The Ferry will cost you from $28 return for the plebs - or a bit more for a posh seat. It  departs 15  minutes past the hour and takes about 25 minutes to get there. The last return ferry is at 5pm. [Book here](https://mona.net.au/stuff-to-do/ferry)
+#### MONA self-guided adventure
 
+![MONA](imgs/Mona-trip2.png)
 
-## C3 Convention Centre Based Activities  
-These events are self-catered with food available to purchase from the fantastic C3 Cafe. 
+The Museum of Old and New Art (MONA) is truly an experience. Gather a group of friends and catch the ferry to MONA and see the sights on the way there. Entry to MONA will cost $39 if you aren’t a Taswegian. [Check out the website](https://mona.net.au/visit) to see if there are any special additional exhibitions to buy tickets to.
 
-### **_OpenStreetMap Workshop_**
-![OSM](/imgs/osm5.png)
+The Ferry will cost you from $28 return for the plebs, or you can splash out for a seat posh seat. It departs 15 minutes past the hour and takes about 25 minutes to get there. The last return ferry is at 5pm. [Book here](https://mona.net.au/stuff-to-do/ferry).
 
-Whether you're new to OpenStreetMap, just dabbled in it or a very active contributor, please join us for an OpenStreetMap workshop where you can:
 
-\- Learn how to contributor to OpenStreetMap
+### C3 Convention Centre Based Activities  
+These events are self-catered with food available to purchase from the fantastic C3 Cafe. 
+
+####  OpenStreetMap Mapathon
 
-\- Learn how to consume OpenStreetMap data
+![OSM](imgs/osm5.png)
 
-\- Work on documentation,
+Whether you're new to OpenStreetMap, just dabbled in it or a very active contributor, please join us for an OpenStreetMap workshop where you can:
 
-\- Or join an organised mapping activity
+* Learn how to contributor to OpenStreetMap
+* Learn how to consume OpenStreetMap data
+* Work on documentation,
+* Or join an organised mapping activity
 
 **What to bring:**
 
 - Your own laptop
 
-### **_Data Cube Hack Day_**
-![Data Cube](/imgs/data-cube.png)
+### Datacube Hackathon
 
-Come and join us for the Data Cube Hack Day! If you don’t know about it, we will introduce you to it!
+![Data Cube](imgs/datacube-hack.png)
```

Comment:
I need to fix how images are rendered - they become too big. You can use this to override the width

```suggestion
<div style="max-width:700px"><img src="imgs/datacube-hack.png" alt="Data Cube" style="width:100%"></div>
```